docs/user/troubleshooting-guides/03-10-dns-mgt/03-10-dns-mgt-connection-refused.md

```diff
@@ -0,0 +1,29 @@
+# Connection refused or timeout
+
+## Symptom
+
+After you have finished all the steps required to [set up your custom domain](../../tutorials/01-10-setup-custom-domain-for-workload.md), you receive the `connection refused` or `connection timeout` error when you try to expose a Service. It shows up when you call the Service endpoint by sending a GET request. The error looks as follows:
+
+```txt
+curl: (7) Failed to connect to httpbin.mydomain.com port 443: Connection refused
+```
+
+## Cause
+
+DNS resolves to an incorrect IP address.
+
+## Remedy
+
+Check if the IP address provided as the value of the **spec.targets** parameter of the DNSEntry custom resource (CR) is the IP address of the Ingress Gateway you are using. To check the Ingress Gateway IP, run:
```

Comment:
```suggestion
Check if the IP address provided as the value of the **spec.targets** parameter of the DNSEntry custom resource is the IP address of the Ingress Gateway you are using. To check the Ingress Gateway IP, run:
```
Custom resource is mentioned only once in this file, so I think we don't need the abbreviation.